onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -4081,25 +4082,27 @@ TEST_F(GraphTransformationTests, FilterEnabledOptimizers) {
 TEST_F(GraphTransformationTests, PropagateCastOpsTests) {
```

Comment:
>GraphTransformationTests [](http://example.com/codeflow?start=7&length=24)

did you measure how much time it will take to run all UTs here?

Reply:
Timed using using the release build:
$ time ./onnxruntime_test_all --gtest_filter=GraphTransformationTests.PropagateCastOpsTests
real    0m0.298s
user    0m0.621s
sys     0m0.089s
Time to run tests doesn't worry me.